pages_/register-phone.tsx

```diff
@@ -285,4 +305,14 @@ const VerifyPhoneWithCaptcha = () => {
   );
 };
 
+export const getServerSideProps: GetServerSideProps = async (context) => {
+  const token = getToken(context);
+
+  return {
+    props: {
+      isLoggedIn: token?.clientId !== undefined,
```

Comment:
why not just check if token isn't undefined? can we have a valid token without a clientId?

Reply:
done